doc/api/cli.md

```diff
@@ -218,22 +218,23 @@ or be removed in the future.
 
 To avoid these cases, any builtin function overrides should be defined upfront:
 
-<!-- eslint-disable no-redeclare -->
 ```js
 const o = {};
 // THROWS: Cannot assign read only property 'toString' of object
 o.toString = () => 'string';
 
-// OK
-const o = { toString: () => 'string' };
-
 class X {
   constructor() {
     this.toString = () => 'string';
   }
 }
 // THROWS: Cannot assign read only property 'toString' of object
 new X();
+```
+
+```js
+// OK
+const o = { toString: () => 'string' };
```

Comment:
Why'd you move this? Any specific reasons?

Reply:
Due to fixes in the parser the former notation resulted in an syntax error.